CMakeLists.txt

```diff
@@ -1759,9 +1780,11 @@ add_hpx_pseudo_target(core)
 ################################################################################
 # Configure compression and other plugins
 ################################################################################
-set(HPX_STATIC_PARCELPORT_PLUGINS "" CACHE INTERNAL "" FORCE)
-add_hpx_pseudo_target(plugins)
-add_subdirectory(plugins)
+if(HPX_WITH_DISTRIBUTED_RUNTIME)
+  set(HPX_STATIC_PARCELPORT_PLUGINS "" CACHE INTERNAL "" FORCE)
+  add_hpx_pseudo_target(plugins)
+  add_subdirectory(plugins)
```

Comment:
Why can't we have plugins in a non distributed world?

Reply:
That's a fair point. I haven't actually disabled the plugin functionality. Disabling the plugins at this level was more for simplicity as none of the plugins we have at the moment make much sense with a local-only runtime. I can go and disable them individually instead.